integration-tests/pom.xml

```diff
@@ -29,6 +29,11 @@
     <description>Integration tests using the httpclient module</description>
 
     <dependencies>
+        <dependency>
```

Comment:
This is incorrect, this SDK should not depend on the Authn SDK even for ITs. Please remove this!